### DISCUSSION TITLE
[FEATURE REQUEST] Using space instead of underscore while writing down the final file name. This way we can copy the prompt from the image directly

### DISCUSSION THREAD

**Discussion by dineshdharme** [hlky/stable-diffusion-webui#366] [Feature suggestions]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by hlky** [comment]
You can copy the output info here
![image](https://user-images.githubusercontent.com/106811348/187511688-1e9eb8d5-3605-4de0-8a19-57df9257bdc9.png)

**Comment by Nyxeka** [comment]
webui.py:770

replace 
`sanitized_prompt = prompts[i].replace(' ', '_').translate({ord(x): '' for x in invalid_filename_chars})`
with
sanitized_prompt = prompts[i].translate({ord(x): '' for x in invalid_filename_chars})

**Comment by hlky** [comment]
@Nyxeka thanks, I'll need to add it as a toggle in the UI, I'm just working on some other things atm

**Comment by dineshdharme** [comment]
I requested this feature because in case we have to go back to the generated images and use that prompt. This will be similar to how Dalle-2 saves images.

**Reply by Nyxeka** [reply]
> I requested this feature because in case we have to go back to the generated images and use that prompt. This will be similar to how Dalle-2 saves images.

you can just tick off the "Write prompt info files". You'll find the sample images actually only save a certain number of characters anyways...

With write prompt sample info files checked, you get a little .yaml next to every image with all of the prompt information, the seed, the sampling algorithm, etc

**Comment by thedyze** [comment]
underscores in the prompt doesn't seem to matter, I use it all the time
